bench/wiredtiger-run-ycsb-c4.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+#
+# 2021-08-19 J. Dagger
+#
+# Execute wtperf on QEMU and then shutdown machine
+
+chmod +x mount_sdb.sh
+./mount_sdb.sh
+
+mv wiredtiger /media/wiredtiger # move to larger shared drive
+cd /media/wiredtiger/build_posix/bench/wtperf
+./wtperf -O ../../../bench/wtperf/runners/ycsb-c4.wtperf
+
+echo MONITOR CSV
+cat WT_TEST/monitor
+echo CSV DONE
+
+poweroff -f
```

Comment:
Why is it important to shut down the machine at the end of the test? This is asymmetric (i.e. you don't start the machine in this script) which can lead to issues when composing scripts later.

Reply:
It's only called from qemu-test.sh which starts the machine. However, I realize now how that implementation could be problematic so I'll try and come up with something better

Reply:
The shutdown is no longer initiated from these scripts. It's now an option in qemu-test.sh